personalized_nlp/models/baseline.py

```diff
@@ -8,6 +8,8 @@ def __init__(self, output_dim=2, text_embedding_dim=768, **kwargs):
         super().__init__()
         self.text_embedding_dim = text_embedding_dim
         self.fc1 = nn.Linear(text_embedding_dim, output_dim) 
+        self.is_frozen = False
```

Comment:
It is currently used because the current train.py returns error, since it always check for the is_frozen value, even when I refer to the baseline_tuned model. I will update the train.py file before removing this line, thank you.